localstack-core/localstack/services/events/event_bus.py

```diff
@@ -45,6 +46,7 @@ def create_event_bus_service(
                 region,
                 account_id,
                 event_source_name,
+                description,
```

Comment:
as a note, I think it would be good to use keyword arguments here, it's much cleaner and readable (named parameters)